addons/survey/models/survey_user.py

```diff
@@ -39,11 +39,14 @@ class SurveyUserInput(models.Model):
     invite_token = fields.Char('Invite token', readonly=True, copy=False)  # no unique constraint, as it identifies a pool of attempts
     partner_id = fields.Many2one('res.partner', string='Partner', readonly=True)
     email = fields.Char('Email', readonly=True)
+    survey_user_nickname = fields.Char('Nickname', help="Attendee nickname, mainly used to identify him in survey sessions rankings.")
```

Comment:
Why "survey_user_nickname" ? Why not simply nickname ?

Reply:
I thought it was easier to grep but I just checked and there are not a lot of "nicknames" in the Odoo.

Renamed it nickname.